File: python/shortened_url.py
class ShortenedURL:
    def __init__(self, url):
        self.hits = 0
        self.url = url

    def __repr__(self):
        return "<ShortenedURL url:%s hits:%s>" % (self.url, self.hits)

    def hit(self):
        self.hits += 1

File: java/src/shortener/ShortenedURL.java
package shortener;

import java.util.concurrent.atomic.AtomicInteger;

public class ShortenedURL {
    private String url;
    private AtomicInteger hits;

    public ShortenedURL(String url) {
        this.url = url;
        this.hits = new AtomicInteger();
    }

    public int hit() {
        return this.hits.incrementAndGet();
    }

    public String getURL() {
        return url;
    }

    public int getHits() {
        return hits.get();
    }
}
